src/views/CategoryPage/CategoryBody/CategoryBody.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ArticleCard from '@Components/ArticleCard';
+import connectComponent from '@Lib/connect-component';
+import './CategoryBody.scss';
+
+export class CategoryBody extends Component {
+  constructor(props) {
+    super(props);
+    this.displayCards = this.displayCards.bind(this);
```

Comment:
Good work here @Lundii , is there a reason you used the bind method on the 'this' keyword since an arrow function would prevent that, or do you just prefer to use bind.

Reply:
It's more of a preference. I normally bind my class methods on the constructor.

Reply:
Okay, that's cool. Good job again.